apps/discovery_api/test/integration/docker-compose.yaml

```diff
@@ -71,13 +84,18 @@ services:
     - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
     - "KAFKA_LISTENERS=INSIDE://:9094,OUTSIDE://:9092"
     - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
-    - "KAFKA_CREATE_TOPICS=event-stream:1:1,dead-letters:1:1"
+    - "KAFKA_CREATE_TOPICS=event-stream:1:1,streaming-transformed:1:1,dead-letters:1:1,streaming-persisted:1:1"
     - "KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181"
     healthcheck:
-      test: ["CMD-SHELL", "kafka-topics.sh --zookeeper zookeeper:2181 --list | grep event-stream || exit 1"]
+      test: ["CMD-SHELL", "cat /opt/kafka/logs/server.log | grep \"Previous Leader Epoch was: -1\""]
       interval: 10s
       timeout: 20s
       retries: 3
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
```

Comment:
Looking forward to using these new images, maybe test startup time will be a bit faster.

Why was the redis healthcheck removed? As well as metastore's. Can the new images for those have healthchecks? I believe divo uses them to surface if there was issues starting them up, which has been helpful in the past.

Reply:
Hmm, a valid question. I haven't noticed issues with startup so far. Maybe as part of a followup card assuming the builds go well?